mappings/net/minecraft/util/Identifier.mapping

```diff
@@ -7,7 +7,7 @@ CLASS net/minecraft/class_2960 net/minecraft/util/Identifier
 	METHOD <init> (Ljava/lang/String;)V
 		COMMENT <p>Takes a string of the form {@code <namespace>:<path>}, for example {@code minecraft:iron_ingot}.</p>
 		COMMENT <p>The string will be split (on the {@code :}) into an identifier with the specified path and namespace.</p>
-		COMMENT Prefer using the {@link #Identifier(java.lang.String, java.lang.String)} constructor that takes the namespace and path as individual parameters to avoid mistakes.
+		COMMENT Prefer using the {@linkplain net.minecraft.util.Identifier#Identifier(java.lang.String, java.lang.String) Identifier(java.lang.String, java.lang.String)} constructor that takes the namespace and path as individual parameters to avoid mistakes.
```

Comment:
I think `link` would be better here than `linkplain` since the link text is code.